1-exercises/D-do-while/exercise.js

```diff
@@ -8,8 +8,20 @@
 
 function evenNumbersSum(n) {
     // TODO
+    let sum = 0;
+    let num = 0;
+    let counter = 0;
+    do {
+        sum += num;
+        num += 2;
+        counter++;
+    
+    } while (counter<n);
+
+    return sum;
 }
 
+
```

Comment:
I think You need the loop to add up the first n even numbers,
starting at 0. So if n=3, you need to add up 0+2+4. If n=10 you
need to add up 0+2+4+6+8+10+12+14+16+18.